practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/filters/ParameterFilter.java

```diff
@@ -0,0 +1,30 @@
+package com.dowa.java.web.filters;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class ParameterFilter implements Filter {
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        System.out.println("ParameterFilter init");
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
+            throws IOException, ServletException{
+        System.out.println("ParameterFilter do filter");
+
+        if (servletRequest instanceof HttpServletRequest) {
+            //StringBuffer requestURL = ((HttpServletRequest) servletRequest).getRequestURL();
+            String name = servletRequest.getParameter("name");
```

Comment:
what if the parameter is not called "name" how would you do this? such that no matter the parameter name the filter would still work?